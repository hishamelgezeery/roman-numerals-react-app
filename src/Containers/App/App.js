import "bootstrap/dist/css/bootstrap.min.css";
import Header from "../../Components/Header/index";
import Converter from "../Converter";

function App() {
  return (
    <div className="App">
      <Header
        navbarLink="#"
        navbarText="RN Converter"
        style={{ backgroundColor: "#FDEECD" }}
        expandType={"lg"}
        variantType={"light"}
      />
      <div className="App">
        <Converter />
      </div>
    </div>
  );
}

export default App;
