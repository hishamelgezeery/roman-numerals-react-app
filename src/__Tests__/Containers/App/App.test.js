import renderer from 'react-test-renderer'
import App from '../../../Containers/App'

describe('App Container', () => {
  it('should render correctly', () => {
    const instance = renderer.create(<App
        />).toJSON()
    expect(instance).toMatchSnapshot()
    })
})
