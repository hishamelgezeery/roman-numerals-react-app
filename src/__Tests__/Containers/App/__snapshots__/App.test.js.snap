// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Container should render correctly 1`] = `
<div
  className="App"
>
  <nav
    className="navbar navbar-expand-lg navbar-light"
    style={
      Object {
        "backgroundColor": "#FDEECD",
      }
    }
  >
    <a
      className="navbar-brand"
      href="#"
    >
      RN Converter
    </a>
  </nav>
  <div
    className="App"
  >
    <div
      className="container container"
    >
      <div
        className="row"
      >
        <div
          className="col-sm-12 col-md-5"
        >
          <h1
            className="centerAlign"
          >
            Roman Numerals converter
          </h1>
          <form
            className=""
          >
            <div
              className="row inputsMargin"
            >
              <div
                className="col-sm-3"
              >
                <label>
                  Roman Number
                </label>
              </div>
              <div
                className="col-sm-9"
              >
                <div
                  className="input-group"
                >
                  <input
                    className="form-control"
                    onInput={[Function]}
                    value=""
                  />
                  <div
                    className="input-group-append"
                  >
                    <button
                      className="btn btn-outline-secondary"
                      disabled={false}
                      onClick={[Function]}
                      type="button"
                    >
                      Convert
                    </button>
                  </div>
                </div>
              </div>
            </div>
            <div
              className="row inputsMargin"
            >
              <div
                className="col-sm-3"
              >
                <label>
                  Decimal Number
                </label>
              </div>
              <div
                className="col-sm-9"
              >
                <div
                  className="input-group"
                >
                  <input
                    className="form-control"
                    onInput={[Function]}
                    value=""
                  />
                  <div
                    className="input-group-append"
                  >
                    <button
                      className="btn btn-outline-secondary"
                      disabled={false}
                      onClick={[Function]}
                      type="button"
                    >
                      Convert
                    </button>
                  </div>
                </div>
              </div>
            </div>
            <div
              className="row inputsMargin"
            >
              <div
                className="col-sm-3"
              >
                <label>
                  Calculation
                </label>
              </div>
              <div
                className="col-sm-9"
              >
                <textarea
                  className="fullWidth"
                  readOnly={true}
                  rows={4}
                  value=""
                />
              </div>
              <div
                className="offset-sm-3 resetContainerStyle"
              >
                <button
                  className="btn btn-outline-secondary"
                  disabled={false}
                  onClick={[Function]}
                  type="button"
                >
                  RESET
                </button>
              </div>
            </div>
          </form>
        </div>
        <div
          className="col-sm-12 col-md-7  d-flex align-items-center"
        >
          <img
            alt="Roman Numerals"
            className="imageStyle"
            src="/roman-numerals.jpeg"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
